src/xo-app/vm/tab-disks.js

```diff
@@ -119,7 +209,6 @@ class NewDisk extends Component {
     const { vm, isAdmin } = this.props
     const { formatMessage } = this.props.intl
     const { size, sr, name, bootable, readOnly } = this.state
-
```

Comment:
Why?

Reply:
Because, the following line contains another initialization with const.